src/core/opamCompat.ml

```diff
@@ -130,3 +130,14 @@ module Stdlib = Pervasives
 #else
 module Stdlib = Stdlib
 #endif
+
+#if OCAML_VERSION >= (4, 13, 0)
+module Lazy = Lazy
+#else
+struct
```

Comment:
```suggestion
module Lazy = struct
```

Reply:
Oops - fixed (slightly differently, w.r.t. to the other modules)